cartography/intel/aws/ecr.py

```diff
@@ -61,46 +61,69 @@ def load_ecr_repositories(neo4j_session, data, region, current_aws_account_id, a
 
 
 @timeit
-def load_ecr_repository_images(neo4j_session, repo_data, region, aws_update_tag):
+def transform_ecr_repository_images(repo_data):
+    """
+    Ensure that we only load ECRImage nodes to the graph if they have a defined imageDigest field.
+    """
+    repo_images_list = []
+    for repo_uri, repo_images in repo_data.items():
+        filtered_imgs = []
+        for img in repo_images:
+            if 'imageDigest' in img and img['imageDigest']:
+                filtered_imgs.append(img)
+            else:
+                logger.warning(
+                    "Repo %s has an image that has no imageDigest. Its tag is %s. Continuing on.",
+                    repo_uri,
+                    img.get('imageTag'),
```

Comment:
I am allowing imageTag to be None. I really don't think this will ever happen, but then again I also didn't think that we'd ever have a case where imageDigest returned None :-)